app/src/shared/web3/config.js

```diff
@@ -33,8 +34,10 @@ const getConfig = (): Config => ({
         address: process.env.DAI_TOKEN_CONTRACT_ADDRESS || '',
     },
     marketplace: {
-        abi: marketplaceAbi,
-        address: process.env.MARKETPLACE_CONTRACT_ADDRESS || '',
+        abi: process.env.NEW_MP_CONTRACT ? marketplaceAbi : marketplaceAbiOld,
+        address: (process.env.NEW_MP_CONTRACT ?
```

Comment:
And as a note, the vars need to be evaluated inside the method instead of the top level because `process.env` vars are overwritten in the test before calling the method but *after* loading this file.

Reply:
Ahh, interesting.